bookserver/routers/auth.py

```diff
@@ -88,11 +88,12 @@ async def login(
         # alg$salt$hash
         # We need to grab the salt and provide that to the CRYPT function
         # which we import from pydal for now.  Once we are completely off of
-        # web2py then this will change.  The ``web2py_private_key`` is an environment
+        # web2py then this will change. The ``web2py_private_key`` is an environment
```

Comment:
Dude!  I've been putting 2 spaces after a period since before you were born!

Reply:
:) [MLA](https://libguides.css.edu/c.php?g=41682&p=265045) and most modern style guide specify one space. But actually [two spaces is better](https://www.theatlantic.com/science/archive/2018/05/two-spaces-after-a-period/559304/)?